x-pack/plugins/endpoint/common/types.ts

```diff
@@ -442,3 +445,125 @@ export type AlertingIndexGetQueryInput = KbnConfigSchemaInputTypeOf<
  * Result of the validated query params when handling alert index requests.
  */
 export type AlertingIndexGetQueryResult = TypeOf<typeof alertingIndexGetQuerySchema>;
+
+/**
+ * Endpoint Policy configuration
+ */
+export interface PolicyConfig {
+  windows: {
+    events: {
+      dll_and_driver_load: boolean;
```

Comment:
were we thinking about making these events keys part of another enum?

Reply:
Not sure what you mean. This was moved to `common` because it was used by (or was related to a type used by) the generator code (which is in common.)